server/service/client.go

```diff
@@ -514,7 +514,7 @@ func (c *Client) ApplyGroup(
 		if len(tmScriptsPayloads) > 0 {
 			for tmName, scripts := range tmScriptsPayloads {
 				if err := c.ApplyTeamScripts(tmName, scripts, opts); err != nil {
-					return fmt.Errorf("applying scripts for team %q: %w", tmName, err)
+					return fmt.Errorf("Couldn't edit scripts. %w", err)
```

Comment:
That's probably what the figma/spec mentioned, but I think we should keep the team name as part of the error to help the user figure out which team's scripts failed - there may be multiple teams in a single yaml file.

Reply:
Makes sense to me!